docs/_form-helpers/date.md

```diff
@@ -47,6 +47,32 @@ data-*      | string | Data attributes, eg: `'data-foo': 'bar'`
 
 Any other options not listed here will be applied to the input.
 
+## US and Reverse Formats
+
+Add the `data-datepickerStyle': 'US'` or `'data-datepickerStyle': 'reverse'` option to format the date properly.
```

Comment:
Can we turn this into a normal `format` option for the helper instead of a data attribute? it can turn into a data attribute in the actual code, but helper options are usually simpler

Reply:
Are these the only options we need to support, or should we simply accept a standard date time formatted string?

Reply:
Based on the wikipedia article and our current needs I would suggest we go with those until there is a feature request for something more complex. We are speaking only about dates and not datetime strings. Will simplify the rest based on your first comment

Reply:
Sorry to be the guy that says "Shouldn't we name this differently", but ... Shouldn't we name this differently? 

I think `datePickerStyle` could be ambiguous in this context, should we consider calling this `datePickerFormat` (to coincide with the wiki) and to remove the possible assumption that this option is tied to styling (appearance / css)

Reply:
Because we are speaking about html attributes, everything that doesn't exist already should be data attribute to future-proof our code. As @mikedevelops mentioned, if in HTML6 for example they decide to add format for other use we will need to change it.

Reply:
For the vast majority of helpers we've gone with simple options, I'd much prefer this to be `format` rather than a data attribute for the helper's options object, please.